docs/content/en/docs/implementing/otel.md

```diff
@@ -4,26 +4,193 @@ description: How to standardize access to OpenTelemetry observability data
 weight: 140
 ---
 
+
+The Keptn Lifecycle Toolkit (KLT) makes any Kubernetes deployment observable.
+In other words, it creates a distributed, end-to-end trace
+of everything Kubernetes does in the context of a Deployment.
+It provides this information
+for all applications running in your cluster,
+and includes information about
+everything Kubernetes does in the context of a deployment.
+To do this,
+Keptn introduces the concept of an `application`,
+which is an abstraction that connects multiple
+Workloads that logically belong together,
+even if they use different deployment strategies.
+
+This means that:
+
+- You can readily see why a deployment takes so long
+  or why it fails, even when using multiple deployment strategies.
+- KLT can capture DORA metrics and expose them as OpenTelemetry metrics
+
+The observability data is an amalgamation of the following:
+
+- DORA metrics are collected out of the box
+  when the Lifecycle Toolkit is enabled
+- OpenTelemetry runs traces that show
+  everything that happens in the Kubernetes cluster
+- Custom Keptn metrics that you can use to monitor
+  information from all the data providers configured in your cluster
+
+All this information can be displayed with dashboard tools
+such as Grafana.
+
+For an introduction to using OpenTelemetry with Keptn metrics, see the
+[Standardize observability](../getting-started/observability)
+getting started guide.
+
+## DORA metrics
+
+DORA metrics are an industry-standard set of measurements;
+see the following for a description:
+
+- [What are DORA Metrics and Why Do They Matter?](https://codeclimate.com/blog/dora-metrics)
+- [Are you an Elite DevOps Performer?
+   Find out with the Four Keys Project](https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance)
+
+DORA metrics provide information such as:
+
+- How many deployments happened in the last six hours?
+- Time between deployments
+- Deployment time between versions
+- Average time between versions.
+
+The Keptn Lifecycle Toolkit starts collecting these metrics
+as soon as you apply
+[basic annotations](integrate/#basic-annotations)
+to the `Deployment` resource.
```

Comment:
I would remove `Deployment` since we support a set of resources: Deployment, ReplicaSet, StatefulSet, Pod, etc
All of these are under the name "Workload": https://kubernetes.io/docs/concepts/workloads/
```suggestion
to the Workload resource.
```